.eslintrc.js

```diff
@@ -0,0 +1,95 @@
+module.exports = {
+    extends: [
+        'eslint:recommended',
+        'plugin:jest/style',
+        'plugin:jest/recommended',
+        'prettier',
+        'plugin:prettier/recommended',
+    ],
+    parserOptions: {
+        ecmaVersion: 2021,
+        sourceType: 'module',
+        ecmaFeatures: {
+            impliedStrict: true,
+        },
+    },
+    ignorePatterns: ['out/**/*'],
+    env: {
+        es2021: true,
+        node: true,
+        es6: true,
+        jest: true,
+    },
+    globals: {
+        Atomics: 'readonly',
+        SharedArrayBuffer: 'readonly',
+    },
+    plugins: ['prettier', 'jest'],
+    overrides: [
+        {
+            files: ['*.ts', '*.tsx'],
+            plugins: ['@typescript-eslint'],
+            parser: '@typescript-eslint/parser',
+            extends: ['plugin:@typescript-eslint/recommended', 'prettier'],
+
+            rules: {
+                'prettier/prettier': 'error',
+                'no-var': ['error'],
+                'prefer-const': ['error'],
+                'no-confusing-arrow': 'error',
+                'no-multi-spaces': ['error'],
+                'no-multi-str': ['error'],
+                'no-param-reassign': ['error'],
+                'no-return-assign': ['error'],
+                'no-return-await': ['error'],
+                'no-throw-literal': ['error'],
+                'no-useless-catch': ['error'],
+                'no-useless-return': ['error'],
+                'no-empty': ['error'],
+                'no-extra-boolean-cast': ['error'],
+                'no-floating-decimal': ['error'],
+                'no-warning-comments': ['error'],
+                'no-irregular-whitespace': ['error'],
+                'no-delete-var': ['error'],
+                'no-undef-init': ['error'],
+                'no-redeclare': ['error'],
+                'no-else-return': [
+                    'error',
+                    {
+                        allowElseIf: false,
+                    },
+                ],
+                curly: ['error', 'all'],
+                semi: ['error', 'always'],
+                eqeqeq: ['error', 'always'],
+                'semi-style': ['error', 'last'],
+                'array-bracket-spacing': ['error'],
+                'linebreak-style': ['error', 'unix'],
+                'callback-return': ['error'],
+                'block-spacing': ['error'],
+                'comma-style': ['error', 'last'],
+                'eol-last': ['error', 'always'],
+                'lines-between-class-members': ['error', 'always'],
+                'max-len': [
+                    'error',
+                    {
+                        code: 120,
```

Comment:
What is the reason for this value? 120 chars is way to long. My suggestion is to keep it between 70 - 80 chars

Reply:
Again a preference thingy. Typescript prints better on a wider width. I have reverted to 80.